include/ignition/gazebo/gui/TmpIface.hh

```diff
@@ -63,15 +63,6 @@ namespace ignition
       /// \param[in] _path Path to world file.
       public slots: void OnSaveWorldAs(const QString &_path);
 
-      /// \brief Server control service callback
-      /// This is the server-side logic which provides the world_control
-      /// service.
-      /// \param[in] _req Request
-      /// \param[out] _res Response
-      /// \return True for success
-      private: bool OnServerControl(const msgs::ServerControl &_req,
-                                          msgs::Boolean &_res);
```

Comment:
This is technically ABI breaking, even if we don't expect anyone to be using this header. Can we leave the header and just remove the implementaiton? The header is already gone in newer versions.

Reply:
You're right, this would be technically an ABI break. I returned the function and marked it deprecated - it's not exactly following the tick-tock deprecation rule, but as it was a private function and I removed the only call to it, it should be pretty much safe.